app/controllers/store_tags_controller.rb

```diff
@@ -3,9 +3,12 @@ class StoreTagsController < ApplicationController
   def create
     tag = Tag.find_by(name: params[:name])
     tag = Tag.create(name: params[:name]) unless tag
-    StoreTag.create(store_id: params[:store_id], tag_id: tag.id)
-    
-    redirect_to "/stores/#{Store.find_by(id: params[:store_id]).yelp_id}"
+    store_tag = StoreTag.create(store_id: params[:store_id], tag_id: tag.id)
+    if request.xhr?
```

Comment:
@Nova840 Would you mind looking up redirect to or whatever Pan mentioned in the code review this afternoon to see if that method would work?